### ISSUE TITLE
Ability to store created twitter walls with the account

### ISSUE THREAD

**Issue by aneeshd16** [loklak/loklak_webclient#102]
* User should be able to create a set of twitter walls with different configurations and have them associated with the user's account.
* Requires support from loklak_server for storage.
* Since the accounts servlet can be accessed only through localhost, this requires support through oauth-proxy

**Comment by Orbiter** [comment]
I propose to store the configuration together with authentication json in a property 'apps'. Thats already implemented! Set apps to a json like
```
apps: {wall:{ .. your wall data }}
```
so every loklak app can add their property too.

**Comment by aneeshd16** [comment]
Perfect! Is there a limit on how much data can be stored? Lets say, if i wanted to store a base64 image inside the JSON, would it be okay?

**Comment by Orbiter** [comment]
There is no limit yet. Should we set one?

**Comment by aneeshd16** [comment]
Other apps (like the ones rendering images of maps) would store any amounts of images with an account. While there is no need for setting a limit now, there could be space constraints later. I am not sure about this, I leave that to you.

**Comment by Orbiter** [comment]
maps shall not be stored in that way but generated on-the-fly using the feature in https://github.com/loklak/loklak_server/issues/39

**Comment by Orbiter** [comment]
The account servlet can now be only accessed from localhost for security reason. Please check if this affects your architecture. Maybe this servlet must be retrieved through a proxy, i.e. the oauth_proxy (if applicable)

**Comment by aneeshd16** [comment]
I'll edit the issue.

**Comment by mariobehling** [comment]
@aneeshd16 Do you have an update here?

**Comment by aneeshd16** [comment]
#102, #103, and #74 are to be implemented together.Can we move this to M2.5?

**Comment by aneeshd16** [comment]
Process to be followed:
````
If user is signed in:
    Get all user data from loklak_server
    Do changes
    Save all user data to loklak_server
````

Incremental updates are not possible as of now due to the nature of the API, so the the entire user data has to be loaded and saved everytime.

**Comment by aneeshd16** [comment]
Closed via 67308b8826bb565873bf95432965f8dacca9c2cd
